### PR TITLE
Add Github Template

### LINKED ISSUE #15: Improve issue/pr template
1. Investigate public and popular libraries
2. See how they manage issue/pr template
3. present to team your idea, that is inspired from them.
4. Implement your idea after team presentation

### PR DESCRIPTION
**Pull request by vahidtwo**

### CODE REVIEW

**Review by ManiMozaffar — APPROVED** [review]
LGTM!

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        - osx
+        - linux
+        - windows
+    validations:
+      required: false
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: Which SDK version?
```
Comment (aerosadegh) [inline .github/ISSUE_TEMPLATE/bug.yml]: SDK?
Library/etc.!

```description: Which Library/FastCrawler version?```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: 💡 Feature Request
+description: Create a feature request for fastcrawler.
+labels: 'enhancement'
```
Comment (aerosadegh) [inline .github/ISSUE_TEMPLATE/feature.yml:3]: does it correct?
or 

```
labels: 
  - 'enhancement'
 ```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 3]
```diff
@@ -6,9 +6,9 @@ body:
     attributes:
       value: Thanks for taking the time to file a feature request! Please fill out this form as completely as possible.
   - type: textarea
-    id: problem
+    id: usecase
     attributes:
-      label: Problem Statement
+      label: Use Case Statement Statement
```
Comment (aerosadegh) [inline .github/ISSUE_TEMPLATE/feature.yml]: Statement Statement?

**Review by aerosadegh — APPROVED** [review]

**Pull request merged**